src/webui/www/private/views/preferences.html

```diff
@@ -577,9 +577,11 @@
                     </td>
                     <td>
                         <select id="max_ratio_act">
-                            <option value="0">QBT_TR(Pause them)QBT_TR[CONTEXT=OptionsDialog]</option>
-                            <option value="1">QBT_TR(Remove them)QBT_TR[CONTEXT=OptionsDialog]</option>
-                            <option value="2">QBT_TR(Enable super seeding for them)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="0">QBT_TR(Pause torrent)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="1">QBT_TR(Remove torrent)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="2">QBT_TR(Remove torrent + files)QBT_TR[CONTEXT=OptionsDialog]</option>
+                            <option value="3">QBT_TR(Remove torrent + folder)QBT_TR[CONTEXT=OptionsDialog]</option>
```

Comment:
I find the differentiation between "files" and "folder" a bit non-obvious. I had to go back and look at the descriptions in the original PR to figure it out. I don't think "folders" and "folders (force)" is super obvious either  (used in the original PR), but I believe it's the same nomenclature that uTorrent uses.

Reply:
Ok, will restore plural form and use "folders (force)".

Reply:
Please take your time, wait for other opinions.

Reply:
Oh ok

Reply:
I still prefer singular form. The meaning is "When (some) torrent reaches seeding time limit pause (delete etc.) it". Plural form sounds like "When (several/all) torrents reache seeding time limit pause (delete etc.) them" which is incorrect, IMO.